synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -445,6 +445,18 @@ export class SynopsysToolsParameter {
       const parsedUrl = url.parse(collectionUri);
       azureInstanceUrl = `${parsedUrl.protocol}//${parsedUrl.host}`;
       azureOrganization = parsedUrl.pathname?.split("/")[1] || "";
+      if (!azureOrganization) {
+        const protocolSeparator = "://";
+        const protocolIndex =
+          collectionUri.indexOf(protocolSeparator) + protocolSeparator.length;
+        const domainIndex = collectionUri.indexOf(".visualstudio.com");
+        if (protocolIndex >= 0 && domainIndex >= 0) {
```

Comment:
I think, if it is a valid uri then index cannot be 0.

Reply:
will fetch from hostname.